index.html

```diff
@@ -56,7 +56,7 @@
 
 								</p>
 								<ul class="icons">
-									<li><a href="https://twitter.com/devopslinks.com" class="icon fa-twitter"><span class="label">Twitter</span></a></li>
+									<li><a href="https://twitter.com/DevOpsLinks" class="icon fa-twitter"><span class="label">Twitter</span></a></li>
 									<li><a href="https://www.facebook.com/devopslinks" class="icon fa-facebook"><span class="label">Facebook</span></a></li>
 									<li><a href="https://github.com/eon01/" class="icon fa-github"><span class="label">GitHub</span></a></li>

```

Comment:
Why not creating a "company" github account for devopslinks and manage the DevOpsLinks repos from in there? Or you think that the project will not grow to need a such move?